champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/sword/BlockToss.java

```diff
@@ -285,11 +305,13 @@ public void collisionCheck() {
                         continue;
                     }
                 } else if (boulder.isThrown()) {
-                    final List<Entity> nearby = referenceEntity.getNearbyEntities(hitBoxSize, hitBoxSize, hitBoxSize);
+                    final List<LivingEntity> nearby = UtilEntity.getNearbyEnemies(caster, referenceEntity.getLocation(), hitBoxSize);
                     nearby.remove(caster);
-                    nearby.removeIf(entity -> !(entity instanceof LivingEntity) || entity instanceof ArmorStand);
+
```

Comment:
Why did you remove the logic that prevents other entities and armor stands from being damaged? This will cause BlockToss to damage Item Frames, etc

Reply:
because it now only checks for livingentity not entity